src/input/readers/ascii/Ascii.cc

```diff
@@ -395,8 +393,10 @@ bool Ascii::DoUpdate() {
             delete[] fields;
             continue;
         }
+        // If there's no error, then it makes sense to report the next error.
```

Comment:
If every other line is invalid this doesn't improve the situation much, but agree this works in the case where few or a lot of consecutive lines are broken.

I do wonder if it would make sense to count failures and output a summary on `StopWarningSuppression()`, a user might at least then have an indication if it was 3 lines failing, or 10000 lines - right now it's pretty noticeable spill of warnings.

```
warning: ..<...>/Input::READER_ASCII: ../input.log, Suppressed 120121 warnings"
```

Reply:
Seems nice to have, I'll experiment with this